architecture-building-systems/CityEnergyAnalyst
# I1052 scenario comparison

@roglen Would you mind testing the graphs? I would give you the case study via slack.

- the steps to test: 
1. delete your config file.
2. unzip the case study in C
3. run cea/plots/plot_main.

- how to check results:
- 21 plots should have been created in ...baseline/outputs/plots//
